com.gopeed.Gopeed.yml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gopeed
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  - /git_repo
+  - /com.gopeed.Gopeed.desktop
+modules:
+  - name: gopeed
+    buildsystem: simple
+    build-commands:
+      - cp -r Gopeed /app/
+      - chmod +x /app/Gopeed/gopeed
+      - mkdir -p /app/bin
+      - ln -s /app/Gopeed/gopeed /app/bin/gopeed
+      - install -Dm644 com.gopeed.Gopeed.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 git_repo/ui/flutter/assets/icon/icon_1024.png "${FLATPAK_DEST}/share/icons/hicolor/1024x1024/apps/${FLATPAK_ID}.png"
+    sources:
+      - type: archive
+        dest: Gopeed
+        url: https://github.com/GopeedLab/gopeed/releases/download/v1.4.4/Gopeed-v1.4.4-linux-amd64-portable.tar.gz
+        sha256: B8BD718A2627EC4A13CC7D9634DA1355F30C5EAAD63B3A92707D9A703E1619C5
```

Comment:
Failed to download sources: module gopeed: Wrong sha256 checksum for Gopeed-v1.4.4-linux-amd64-portable.tar.gz, expected "B8BD718A2627EC4A13CC7D9634DA1355F30C5EAAD63B3A92707D9A703E1619C5", was "b8bd718a2627ec4a13cc7d9634da1355f30c5eaad63b3a92707d9a703e1619c5"

Reply:
that's the build failure